routes/index/index.hbs

```diff
@@ -1,5 +1,5 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" data-adapt-framework="{{adapt_framework}}" data-adapt-authoring="{{adapt_authoring}}">
```

Comment:
Please can we stick with `data-adapt-framework-version` etc., for alignment with the [framework](https://github.com/adaptlearning/adapt_framework/blob/1cf22d9cb4dae444723f69bff52f3f3883d3d3d9/src/core/js/app.js#L209)?